docs/site/Community-examples.md

```diff
@@ -0,0 +1,82 @@
+---
+lang: en
+title: 'Community examples'
+keywords:
+  LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Examples, Community
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Community-examples.html
+---
+
+In addition to the examples that LoopBack provides and maintains, this page
+lists the examples created by the community.
+
+{% include warning.html content="The examples listed here are not supported by the LoopBack team; they are maintained by the LoopBack community and are listed here for convenience."%}
+
+{% include note.html content="If you want to share your examples in this page, feel free to submit a PR in the [loopback-next repo](https://github.com/strongloop/loopback-next)."%}
```

Comment:
submit a PR to update https://github.com/strongloop/loopback-next/blob/master/docs/site/Community-examples.md?